util/src/main/java/org/datacommons/util/TmcfCsvParser.java

```diff
@@ -320,6 +320,11 @@ public void process(CSVRecord dataRow) {
           ssArg.stripEnclosingQuotes = false;
           // TODO: set stripEscapesBeforeQuotes
```

Comment:
I wonder we might need to do this TODO?

Reply:
I don't think its needed for this change because the escapes are already being stripped